code/modules/fishing/fish/types/ruins.dm

```diff
@@ -98,3 +152,29 @@
 
 /obj/item/fish/skin_crab/get_fish_taste_cooked()
 	return list("cooked crab" = 2)
+
+/obj/item/fish/skin_crab/suicide_act(mob/living/carbon/human/user)
+	user.visible_message(span_suicide("[user] puts [user.p_their()] hand on [src] and focuses intently! It looks like they're trying to transfer their skin to [src]!"))
+	if(!ishuman(user))
+		user.visible_message(span_suicide("[user] has no skin! How embarrassing!"))
+		return SHAME
```

Comment:
Skeletons have no skin and they're human

Reply:
YOU have no skin